### ISSUE TITLE
Browser chrome is hard to distinguish from light web content

### ISSUE THREAD

**Issue by st3fan** [mozilla-mobile/fenix#611]
I am having trouble seeing where the browser chrome (at the bottom of the screen) ends and where page content starts. The color of the chrome is so light that is all blurs together for me.

It looks like Firefox iOS has a slightly darker tint and also has a small darker border at the top of the bottom chrome, which seems to make a big difference.

**Comment by topotropic** [comment]
@shorlander I think that's a valid point – can you have a look?

**Comment by vesta0** [comment]
@lime124  do we have UX feedback on this?

**Comment by shorlander** [comment]
> @shorlander I think that's a valid point – can you have a look?

Yes, the design calls for a border and a shadow to address this:

![Quick Action Buttons](https://user-images.githubusercontent.com/684427/55368881-17ba8880-54c2-11e9-8b73-060c676ff36c.png)

**Comment by yoasif** [comment]
@ekager I know you did a version of this in  #1798 but it looks like the border placement per UX is incorrect, and it also looks like there is a shadow missing. 

Sorry - clearly my bug was a dupe. :(

**Comment by ekager** [comment]
@AmyYLee can this be closed?

**Comment by vesta0** [comment]
@shorlander can we close this?

**Comment by shorlander** [comment]
Yes, this is fixed. Thank you!
